tests/src/test/scala/spire/laws/ExtraLawTests.scala

```diff
@@ -9,7 +9,7 @@ class ExtraLawTests extends FunSuite with Discipline {
   import IntervalSeqArbitrary._
   // import IntervalTrieArbitrary._
 
-  checkAll("Bool[IntervalSeq[Int]]", LogicLaws[IntervalSeq[Int]].bool)
+//  checkAll("Bool[IntervalSeq[Int]]", LogicLaws[IntervalSeq[Int]].bool)
```

Comment:
Thanks for your contribution. Why did you have to comment out this test? Was it failing?